src/zappy_server_src/core/include/rcodes.h

```diff
@@ -0,0 +1,123 @@
+/*
+** EPITECH PROJECT, 2022
+** my_teams
+** File description:
+** rcodes
+*/
+
+/// \file src/zappy_server_src/core/include/rcodes.h
+
+#ifndef RCODES_H_
+    #define RCODES_H_
+
+    #include <stdlib.h>
+    #include <string.h>
+    #include <stdio.h>
+    #include "fd_set_manage.h"
+
+/// \brief The retcodes_s structure contains the representation of the return
+/// code as char *, an int which represent the return code, and a array of
+/// char * which contains the arguments to be used.
+typedef struct retcodes_s {
+
+    /// \brief char * The name of the return code.
+    char *repr;
+
+    /// \brief int The return code.
+    int code;
+
+    /// \brief char ** The arguments of the return code.
+    char **params;
+} retcodes_t;
+
+/// \brief The retcodes_s structure contains the representation of the return
+static const retcodes_t retcodes[] = {
+    {.repr = "200 Success\n",
+    .code = 200,
+    .params = NULL},
+    {.repr = "201 Authentification succeed as : %s\n",
+    .code = 201,
+    .params = NULL},
+    {.repr = "202 Logout succeed as : %s\n",
+    .code = 202,
+    .params = NULL},
+    {.repr = "402 Authentification failed as : %s\n",
+    .code = 402,
+    .params = NULL},
+    {.repr = "501 Command not found : %s\n",
+    .code = 501,
+    .params = NULL},
+    {.repr = "502 Invalid format : %s\n",
+    .code = 502,
+    .params = NULL},
+    {0, 0, 0}
+};
+
+/// \brief Get a retcodes_t structure depending on the code passed as
+/// parameter.
+/// \param int The code to be send.
+/// \return A new instance of retcodes_t.
+static inline retcodes_t *create_new_repcode(int code)
```

Comment:
Inline ? Function is like 15 lines long

Reply:
There is no real regimentation for the use of static inline, so why not